app/src/main/java/com/asfoundation/wallet/repository/GamesRepositoryType.kt

```diff
@@ -0,0 +1,10 @@
+package com.asfoundation.wallet.repository
+
+import com.appcoins.wallet.ui.widgets.GameData
+import io.reactivex.Single
+
+interface GamesRepositoryType {
```

Comment:
I think this nomenclature is not so good because it is different of the usual in android development, I think we should follow.
I like to follow this recommendation:
Names for the implementations of interfaces should be meaningful. Have Default as the prefix if a better name cannot be found. For example, for a NewsRepository interface, you could have an OfflineFirstNewsRepository, or InMemoryNewsRepository. If you can find no good name, then use DefaultNewsRepository. Fake implementations should be prefixed with Fake, as in FakeAuthorsRepository.

more here: https://developer.android.com/topic/architecture/recommendations